cfy_manager/components/postgresql_server/config/patroni.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Runners to orchestrate a high-availability PostgreSQL
+After=syslog.target
+After=network.target
+
+[Service]
+Type=simple
+User=postgres
+Group=postgres
+ExecStart=/opt/patroni/bin/patroni /etc/patroni.conf
+KillMode=process
+TimeoutSec=30
+Restart=no
```

Comment:
no restart?

Reply:
It's actually the default, and was in the recommended patroni service file, so I went with it.

Reply:
Thinking out loud:
Patroni itself won't use much memory, so is unlikely to be an OOM target (which I think tends to be the most frequent cause of process death?) especially sitting next to postgres.
Therefore, if we change this it'll mean systemd will bounce the service if it dies unexpectedly, which we might want, but which I've not observed happen during the torturous tests I put it through.
If we do change it, there'll be slight modifications required on the tests to stop the service restarting itself.

What do you think? Worth changing or not?

Reply:
Changed, probably worth doing to catch unexpected errors. I'll add a comment to the tests jira noting that the kill will also need to stop it being resurrected.